README_Docker.md

```diff
@@ -9,6 +9,10 @@ $apt-cache policy docker-engine
 $sudo apt-get install -y docker-engine
 Docker should now be installed and ready to use
 
+For proxy users, please also make the following change
+https://docs.docker.com/engine/admin/systemd/#http-proxy (HTTP proxy section)
+Create a common network for the containers
+
 Create a common network for the containers
 ------------------------------------------
 $docker network create node-dc-net
```

Comment:
Also it allows for easier copy paste if someone is trying to follow the instructions.